extensions/opentelemetry/opentelemetry/deployment/src/test/java/io/quarkus/opentelemetry/deployment/binders/GrpcOpenTelemetryTest.java

```diff
@@ -32,8 +32,27 @@
 import io.opentelemetry.sdk.trace.data.SpanData;
 import io.quarkus.grpc.GrpcClient;
 import io.quarkus.grpc.GrpcService;
+import io.quarkus.opentelemetry.deployment.Greeter;
+import io.quarkus.opentelemetry.deployment.GreeterBean;
+import io.quarkus.opentelemetry.deployment.GreeterClient;
+import io.quarkus.opentelemetry.deployment.GreeterGrpc;
+import io.quarkus.opentelemetry.deployment.HelloProto;
+import io.quarkus.opentelemetry.deployment.HelloReply;
+import io.quarkus.opentelemetry.deployment.HelloReplyOrBuilder;
+import io.quarkus.opentelemetry.deployment.HelloRequest;
+import io.quarkus.opentelemetry.deployment.HelloRequestOrBuilder;
+import io.quarkus.opentelemetry.deployment.Item;
+import io.quarkus.opentelemetry.deployment.ItemOrBuilder;
+import io.quarkus.opentelemetry.deployment.MutinyGreeterGrpc;
 import io.quarkus.opentelemetry.deployment.MutinyGreeterGrpc.MutinyGreeterStub;
+import io.quarkus.opentelemetry.deployment.MutinyStreamingGrpc;
 import io.quarkus.opentelemetry.deployment.MutinyStreamingGrpc.MutinyStreamingStub;
+import io.quarkus.opentelemetry.deployment.Streaming;
+import io.quarkus.opentelemetry.deployment.StreamingBean;
+import io.quarkus.opentelemetry.deployment.StreamingClient;
+import io.quarkus.opentelemetry.deployment.StreamingGrpc;
+import io.quarkus.opentelemetry.deployment.StreamingProto;
+import io.quarkus.opentelemetry.deployment.common.TestSpanExporter;
```

Comment:
If we are changing the test package, we should also change the package generation of the proto files.

Reply:
> Abound the binder rename, I'm not sure if this would be the right move. The binder concept is something coming from Micrometer and not OpenTelemetry.

He said he was aligning with micrometer... so Bruno definitely did that deliberately.

Binder to me (as a word) implies the glue that links two things together. I personally am not as much of a fan of "instrumenter", though I can understand why OTel picked it. I could be pursuaded either way (I don't have a strong preference), but I will say that the micrometer naming is more intuitive to me.

Reply:
I think what's important is the alignment of concepts and make sure binder/instrumenter packages can be clearly seen. Makes the project simpler.
In relation to using instrumenter... @radcortez has a point on using instrumenter and align with the OTel naming, even if binder is better. Consistency is more important here.

Reply:
Thanks :)

I believe we mixed some conversations here. The original comment for this thread was to align the package generation name of the gRPC proto files as well. Chech here: https://github.com/quarkusio/quarkus/tree/b331569dce285ca4ae976d4d232562e3dc72e934/extensions/opentelemetry/opentelemetry/deployment/src/test/proto.

The binder vs intrumenter was a general comment :)

Reply:
I see. Changed the instrumentation package name nevertheless.  
For the Grpc tests I think proto/ should be inside common/ because they have a similar purpose to the REST endpoints in there.